### DISCUSSION TITLE
Puppeteer support for Arm chips

### DISCUSSION THREAD

**Discussion by ijpatricio** [serversideup/docker-php#84] [General]
Hello there!

I could not find it by any search, and was wondering if you're already using puppeteer by any chance?

I found this image when trying to debug how to put laravel sail and spatie/browsershot (puppeteer under the hood) working together.

**Comment by jaydrogers** [comment]
I haven't used pupeteer before, but it seems like a cool project.

I was thinking of adding some advanced testing with my images and this looks like it could be a cool resource. Thanks for sharing!

**Comment by ijpatricio** [comment]
Glad I could share!

Don't forget to check https://spatie.be/docs/browsershot/v2/introduction
You can easily use it to generate PDFs, images, and more.

There's also https://playwright.dev

Looking forward to checking your work further; looks promising!! 😀
